Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -200,10 +219,10 @@ final class ChatViewController: SLKTextViewController {
     }
 
     override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
-        super.viewWillTransition(to: size, with: coordinator)
-
-        coordinator.animate(alongsideTransition: nil, completion: { _ in
-            self.collectionView?.collectionViewLayout.invalidateLayout()
+        dataController.invalidateLayout(for: nil)
```

Comment:
It was not clear that `nil` would invalidate the entire cache until I double checked the implementation of `invalidateLayout(for:)`. Let's improve it.